uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -33,17 +33,23 @@ <h4 class="modal-title">
       </ui-select>
     </div>
     <div
-      ng-class="{'chr_disabled': (!$ctrl.request.contact_id || $ctrl.postContactSelection)}"
+      ng-class="{'chr_disabled': (!$ctrl.request.contact_id || $ctrl.absenceTypes.length === 0 || $ctrl.postContactSelection)}"
       ng-show="!$ctrl.loading.absenceTypes">
       <div class="row">
         <div class="col-xs-12">
           <div class="chr_custom-select chr_custom-select--full">
             <select
+              ng-hide="$ctrl.absenceTypes.length === 0"
               ng-disabled="$ctrl.isMode('view') || ($ctrl.isRole('manager') && !$ctrl.isMode('create'))"
               name="absenceTypeSelect"
               ng-options="absenceType.id as absenceType.title for absenceType in $ctrl.absenceTypes"
               ng-model="$ctrl.request.type_id"
               ng-change="$ctrl.updateBalance()"></select>
+            <select
+              ng-show="$ctrl.absenceTypes.length === 0"
+              disabled="disabled">
```

Comment:
This select box is used to only show the error message, so it is always disabled and does not depend on a condition. Its visibility is dependent on a condition, which is specified.